Strategic-Initiatives.md

```diff
@@ -11,20 +11,20 @@ and have the support needed.
 
 # Current Initiatives
 
-| Initiative        | Champion                        | Links                                                            |
-|-------------------|---------------------------------|------------------------------------------------------------------|
-| http2             | James Snell                     |                                                                  |
-| Modules           | Myles Borins                    | https://github.com/nodejs/node-eps/blob/master/002-es-modules.md |
-| N-API             | Michael Dawson                  | https://github.com/nodejs/abi-stable-node                        |
-| OpenSSL Evolution | Rod Vagg                        | https://github.com/nodejs/TSC/issues/364                         |
-| Workers           | Anna Henningson                 | https://github.com/nodejs/worker                                 |
-| Error Messages    | James Snell                     | https://github.com/nodejs/node/issues/11273                      |
-| Core Promise APIs | James Snell                     |                                                                  |
-| TSC Governance    | Myles Borins                    | https://github.com/nodejs/TSC/issues/383                         |
-| New Streams APIs  | James Snell + Jeremiah Senkpiel | https://github.com/nodejs/node/pull/16414                        |
-| Moderation        | Rich Trott                      |                                                                  |
-| Async Hooks       | Trevor Norris                   |                                                                  |
-| V8 Currency       | Michaël Zasso                   |                                                                  |
+| Initiative        | Champion                                                  | Links                                                            |
+|-------------------|-----------------------------------------------------------|------------------------------------------------------------------|
+| http2             | [James Snell][jasnell]                                    |                                                                  |
```

Comment:
Maybe add the handle to the text as well in case people are looking for handles to mention? Like `James Snell ([@jasnell][])`